ebms-provider/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -105,17 +107,15 @@ fun Application.ebmsProviderModule() {
     installMicrometerRegistry(appMicrometerRegistry)
     installRequestTimerPlugin()
 
+    install(ContentNegotiation) {
+        json()
```

Comment:
Det er mulig at dette vil påvirke synkron endepunkten og den kan stoppe å fungere.

Reply:
Testet litt lokalt, det burde ikke påvirke annet API fordi de ikke har hatt behov for slik. Derimot i reverse (ved å ta den vekk) kunne det bli problemer med serialisering.